rules/mastg-android-weak-encryption-modes.yaml

```diff
@@ -0,0 +1,16 @@
+rules:
+  - id: weak-encryption-modes
+    languages:
+      - java
+    severity: WARNING
+    message: Weak encryption modes
+    pattern-either:
+      - pattern: Cipher.getInstance("AES/ECB/NoPadding")
+      - pattern: Cipher.getInstance("AES")
+      - pattern: Cipher.getInstance("AES/ECB/PKCS5Padding")
+      - pattern: Cipher.getInstance("AES/ECB/ISO10126Padding")
+      - pattern: Cipher.getInstance("DES/ECB/PKCS5Padding")
+      - pattern: Cipher.getInstance("DESede/ECB/PKCS5Padding")
+    metadata:
+      summary: This rule looks for weak encryption mode like AES/ECB/NoPadding,
```

Comment:
Update to be one line:

This rule looks for weak encryption modes such as AES-ECB.